### PR TITLE
Add the config for circleci syestem.

### PR DESCRIPTION
**Pull request by hwchiu**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12.12.0
```
Comment (ccascone) [inline .circleci/config.yml]: We agreed to stick with node 10

**Comment by teone** [comment]
We need to get this merged or no other PR will pass validation

File: .circleci/config.yml (hunk 1 above)
Comment (hwchiu) [reply .circleci/config.yml]: Done

**Comment by teone** [comment]
+1

**Pull request merged**